Detectors/TRD/base/macros/OCDB2CCDB.C

```diff
@@ -214,7 +214,7 @@ void UnpackGainTable(std::string& gainkey, CalOnlineGainTables* gtbl)
 }
 
 //__________________________________________________________________________________________
-void OCDB2CCDB(Int_t run, const Char_t* storageURI = "alien://folder=/alice/data/2010/OCDB/")
+void OCDB2CCDB(Bool_t writeToLocalCCDB = kTRUE, Int_t run = 297595, const Char_t* storageURI = "local:///cvmfs/alice-ocdb.cern.ch/calibration/data/2018/OCDB")
```

Comment:
Wouldn't it be more flexible to pass the ccdbPath (which is actually a URI) instead of a boolean variable? The dafault could be the test CCDB.

Reply:
Sure, I changed this now to take the CCDB path directly as parameter